### DISCUSSION TITLE
"can't parse element or specie" Problem

### DISCUSSION THREAD

**Discussion by anuri32474** [CederGroupHub/smol#505] [Q&A]
Hello, I need your help. I was trying to use your code, the lmo-drx-ce-mc.ipynb one, and use your drx_prim and drx_entries as examples. but when I tried to specify the cluster subspace, I got an error about failing to parse element or specie :
" Can't parse Element or Species from Li+0.33333333 Mn2+0.33333333 vacA0+0.33333333 " 
I would appreciate your help with this issue. Thank you very much for your consideration.

**Comment by qchempku2017** [comment]
Hi, sorry for not being responsive for long. Luis @lbluque and I @qchempku2017 have not been attending to this package for nearly a year, and your issue could have been caused by recent updates in pymatgen. A quick hacky fix could be creating a separate conda environment while downgrading your pymatgen to 2023.08.10, and your numpy to <2.0 (for example, 1.26.4). 

We are working on an ultimate fix with PR #509 , and we hope to get it released ASAP. Sorry for your inconvenience!
